RMSD.py

```diff
@@ -19,88 +19,120 @@
+    ifsRef = oechem.oemolistream(ref_mol2)
+    print ("Opening reference molecule:", ref_mol2)
 
     # check if the file exist
-    if not ifsRef.open(refFile):
-        print ("Unable to locate %s. Skipping." % refFile)
+    if not ifsRef.open(ref_mol2):
+        print ("Unable to locate %s. Skipping." % ref_mol2.split('/')[-1])
 
     # open query molecule
-    queryFile = ("/work/cluster/nthi/ForceField-Comparison/%s/%s/%s" % (molSet, ffList, molName) )
-    print ("Opening query molecule: ", queryFile)   
-    if os.path.exists(queryFile):
-        ifsQuery = oechem.oemolistream(queryFile)
-   
-        # set flavor for the input file
+    #queryFile = ("/work/cluster/nthi/ForceField-Comparison/%s/%s/%s" % (homeDir, ffList, fName) )
+    print ("Opening query molecule: ", query_mol2  )
+    if os.path.exists(query_mol2):
+        ifsQuery = oechem.oemolistream(query_mol2)
+    # set flavor for the input file
         flavor = oechem.OEIFlavor_Generic_Default | oechem.OEIFlavor_MOL2_Default | oechem.OEIFlavor_MOL2_Forcefield
         ifsRef.SetFlavor(oechem.OEFormat_MOL2, flavor)
         ifsQuery.SetFlavor(oechem.OEFormat_MOL2, flavor)
-   
+
         # create "blank" object
         rmol = oechem.OEGraphMol()
         qmol = oechem.OEGraphMol()
-   
+
         # load molecule from files
         oechem.OEReadMolecule(ifsRef,rmol)
         oechem.OEReadMolecule(ifsQuery,qmol)
-       
+
         # rmsd calculate
         rms = oechem.OERMSD(rmol,qmol, True, True , True)
-        molNames = rmol.GetTitle()
-        logFile.write("%s\t%s\t%s\t%.3e\n" % (molNames, ffRef,ffList, rms) )
-       
-        # different SMILE strings detected
-        if rms == -1:
-            oechem.OEThrow.Warning("Negative RMSD's value detected for %s-%s" % (molNames, ffList) )  
-            nValue.write("%s\t%s\t%s\t%.3e\n" % (molNames, ffRef, ffList, rms) ) 
-
-    # write mol2 file that does not exist into a file
-    else: 
-        oechem.OEThrow.Warning("Unable to locate %s. Skipping." % queryFile)
-        print ("Unable to locate %s. Skipping." % molName)
-        errFile.write("This queryMol does not exist: %s\t%s\t%s\tNaN\n" % (ffList,molSet,molName) )
-    errFile.close()
-    nValue.close()
-    logFile.close()
+    else:
+        rms == -2
+    return rms        
+        
+#----------------------------Parse Inputs-----------------------------------#
 
-#------------------------------Script----------------------------------------#
+if __name__ == '__main__':
+    from optparse import OptionParser
 
-refFile = '/work/cluster/nthi/ForceField-Comparison/DrugBank/SMIRFF'
-refMols = os.listdir(refFile)
+    parser = OptionParser()
+    parser.add_option('-r','--ref',
+            help = "Name of the reference force field",
+            type = "string",
+            dest = 'ref')
 
-listFFs = ['OPLS2005','OPLS3','GAFF2','GAFF']
+    parser.add_option('-c','--compare',
+            help = "Name of the compared force field",
```

Comment:
It might be more accurate to say something along the lines of "Name(s) of the force field(s) to be compared. Can be a single entry or a list of FFs separated by commas."